mne_bids_pipeline/steps/preprocessing/_03_maxfilter.py

```diff
@@ -143,11 +313,6 @@ def run_maxwell_filter(
     )
     bids_path_out = bids_path_in.copy().update(**bids_path_out_kwargs)
 
-    # Now take everything from the bids_path_in and overwrite the parameters
-    subject = bids_path_in.subject  # noqa: F841
-    session = bids_path_in.session  # noqa: F841
-    run = bids_path_in.run
-
```

Comment:
This fixes a logging bug where it used to look like this:
```
[20:05:28] │ ⏳️ preprocessing/_03_maxfilter sub-01 Saving report: /home/larsoner/mne_data/derivatives/mne-bids-pipeline/ds003392/sub-01/meg/sub-01_task-localizer_report.html
[20:05:28] │ ⏳️ preprocessing/_03_maxfilter sub-emptyroom ses-19111211 run-noise Loading reference run: None.
```
instead of this:
```
[20:12:13] │ ⏳️ preprocessing/_03_maxfilter sub-01 Saving report: /home/larsoner/mne_data/derivatives/mne-bids-pipeline/ds003392/sub-01/meg/sub-01_task-localizer_report.html
[20:12:13] │ ⏳️ preprocessing/_03_maxfilter sub-01 run-noise Loading reference run: None.
```
The logging of which file/step we're on makes more sense now I think, and is also consistent with how `_01_data_quality` and `_04_frequency_filter` were already logging:
```
[20:11:40] │ ⏳️ preprocessing/_01_data_quality sub-01 Saving report: /home/larsoner/mne_data/derivatives/mne-bids-pipeline/ds003392/sub-01/meg/sub-01_task-localizer_report.html
[20:11:40] │ ⏳️ preprocessing/_01_data_quality sub-01 run-noise Finding flat channels and noisy channels using Maxwell filtering.
...
[20:12:37] │ ⏳️ preprocessing/_04_frequency_filter sub-01 Saving report: /home/larsoner/mne_data/derivatives/mne-bids-pipeline/ds003392/sub-01/meg/sub-01_task-localizer_report.html
[20:12:38] │ ⏳️ preprocessing/_04_frequency_filter sub-01 run-noise Reading empty-room recording: sub-01_task-noise_proc-sss_raw.fif
```